src/main/kotlin/ru/buhinder/alcoholicservice/service/AlcoholicService.kt

```diff
@@ -23,4 +24,8 @@ class AlcoholicService(
             .map { conversionService.convert(it, AlcoholicResponse::class.java)!! }
     }
 
+    fun getByIds(alcoholicIds: List<UUID>): Flux<AlcoholicResponse> {
```

Comment:
get is used to indicate that the method can throw exception if nothing found and find means that it returns something or empty mono